guides/user/plugin-users/index.md

```diff
@@ -11,78 +11,191 @@ sidebar:
   <strong>Note:</strong> Plugins are an early alpha feature that is under active development and will likely change.
 </div>
 
-This guide is for adding existing plugins to Spinnaker. For information about how to develop a new plugin, see [Plugin Creators Guide](/guides/developer/plugin-creators). 
+In this guide, you add an existing plugin from an [example repository](https://github.com/spinnaker-plugin-examples/examplePluginRepository) to Spinnaker. See the [Plugin Creators Guide](/guides/developer/plugin-creators) for how to create a PF4J or Spring plugin.
+
+# Requirements
+
+* The plugin is either a PF4J or a Spring plugin
+* The plugin resides in a publicly accessible place
+* Spinnaker v1.19 or later
+* Halyard v1.32.0 or later
+* You redeploy Spinnaker using Halyard to apply changes
+
+# Define Plugins
+
+Define plugins in a file called `plugins.json`. This guide uses the  [file](https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json) in the example repository.
+
+```json
+[
+ {
+   "id": <plugin-id>,
+   "description": <description>,
+   "provider": <provider>,
+   "releases": [
+	 {
+	   "version": <version>,
+	   "date": <date>,
+	   "requires": <comma-delimited-list-of-spinnaker-services>,
+	   "sha512sum": <checksum>,
+	   "state": <state>,
+	   "url": <complete-url-to-bundle-zip-file>
+	 }
+   ]
+ }
+]
+```
+
+1. **id**: defined by the plugin creator; find this value in the bundle's `MANIFEST.MF`; unique value
+2. **description**: what the plugin does
+3. **provider**: plugin provider name
+4. **releases**: one-to-many list; each release entry requires:
+
+	* **version**: plugin version (semantic version format, e.g. 1.2.3)
+	* **date**: plugin release date in ISO or yyyy-MM-dd format
+	* **requires**: list of comma-delimited Spinnaker services; ex: "orca>=0.0.0,deck>=0.0.0"
+	* **sha512sum**: a string of the SHA-512 HEX value or the URL to the SHA-512 file
+	* **state**: plugin state, usually "RELEASE"
+	* **url**: either an absolute or relative URL to the bundle zip file
+
+
+# Define the Plugin Repository
+
+Define plugin repositories in a file called `repositories.json`. Each repository exposes the plugins in the corresponding `plugins.json` file to Spinnaker.
```

Comment:
In the heading, you say "plugin repository," singular," but here "Define plugin repositories." And the filename implies plural. 
I would define multiple repositories in one json file? 
Also, is it one repository per plugin, or else why am I defining multiple repositories?

Reply:
Good questions. Yes, you can define one-to-many repositories in `repositories.json`.  Each entry in `repositories.json` points to a single `plugins.json` file in which you define one-to-many plugins. PF4J defines the structure and relationship of those files in the [pf4j-update](https://github.com/pf4j/pf4j-update) project.    
According to the pf4j-update doc, repositories can be local or remote, URL or folder.


```json
[
  {
    "id": "stagingPlugins",
    "url": "http://localhost:8081/plugins.json"
  },
  {
    "id": "myTestPlugins",
    "url": "file:/home/plugins/plugins.json"
  } ,
 {
      "id": "ArmoryExamplePlugins",
      "url": "https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json"
    } 
]
```

Does that clarify why you need both files? If so, I'll update the PR.

Elsewhere in the docs, I need to change the bit about plugins being "publicly accessible" to "accessible by Spinnaker".

Reply:
Added an Overview section